website/docs/r/private_endpoint.html.markdown

```diff
@@ -239,6 +251,16 @@ A `private_service_connection` block exports:
 
 ---
 
+An `ip_configuration` block exports:
```

Comment:
it looks like this has been added to the docs twice?

Reply:
Hi @catriona-m isn't it one for set and the other ones which values you get back?